wai-logger/Network/Wai/Logger/Apache.hs

```diff
@@ -62,6 +75,11 @@ apacheLogStr ipsrc tmstr req status msize =
   <> toLogStr (fromMaybe "" mua)
   <> "\"\n"
   where
+    path =  case pathType of
+      LogOnlyPath ->
+        rawPathInfo req
+      LogPathWithQueryString ->
+        rawPathInfo req <> rawQueryString req
```

Comment:
my idea was to propose backwards-compatible change but if it's OK to change logging format I'll amend the PR